core/src/avm2/globals/flash/display3D/Context3DWrapMode.as

```diff
@@ -6,18 +6,21 @@
 package flash.display3D
 {
 
+    [API("686")]
     public final class Context3DWrapMode
     {
         // Clamp texture coordinates outside the 0..1 range.
         public static const CLAMP:String = "clamp";
 
         // Clamp in U axis but Repeat in V axis.
+        [API("696")]
         public static const CLAMP_U_REPEAT_V:String = "clamp_u_repeat_v";
 
         // Repeat (tile) texture coordinates outside the 0..1 range.
         public static const REPEAT:String = "repeat";
 
         // Repeat in U axis but Clamp in V axis.
+        [API("696")]
         public static const REPEAT_U_CLAMP_V:String = "repeat_u_clamp_v";
```

Comment:
The class-level annotation looks right, but the constants don't seem to have their Flash Player versions mentioned in the docs:
https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/display3D/Context3DWrapMode.html